docs/dev/setup/programming-exercises.rst

```diff
@@ -0,0 +1,59 @@
+Adjustments for programming exercises
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+There are several variables that can be configured when using programming exercises.
+They are present in this separate page to keep the 'normal' setup guide shorter.
+
+
+Path variables
+##############
+
+There are variables for several paths:
+
+- ``artemis.repo-clone-path``
+
+  Repositories that the Artemis server needs are stored in this folder.
+  This e.g. affects repositories from students which use the online code editor or the template/solution repositories of new exercises, as they are pushed to the VCS after modification.
+
+  Files in this directory are usually not critical, as the latest pushed version of these repositories is also stored at the VCS.
+  However, changed that are saved in the online code editor but not yet commited will be lost when this folder is deleted.
+
+- ``artemis.repo-clone-path``
```

Comment:
This is the same as in line 13. Should it be `artemis.repo-download-clone-path`?

Reply:
Good catch! Exactly, it should be ``artemis.repo-download-clone-path``. I changed it